app/assets/stylesheets/welcome.scss

```diff
@@ -1,29 +1,34 @@
-// .vidcontainer {
-//   position: absolute;
-//   width: 100%;
-//   top: 50%;
-//   left: 50%;
-//   margin: -100px 0 0 0;
-// }
+.vidcontainer {
+  position: absolute;
+  width: 100%;
+  top: 50%;
+  left: 50%;
+  margin: 0 0 0 0;
+}
 
 video {
-  position: relative;
+  position: fixed;
```

Comment:
Properties should be ordered background-size, height, left, margin-top, position, -moz-transform, -ms-transform, -webkit-transform, transform, width, z-index